modules/configuration-annotation-processor/src/test/resources/org/apache/ignite/configuration/processor/internal/TestConfigurationSchema.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.ignite.configuration.processor.internal;
+
+import org.apache.ignite.configuration.internal.annotation.Config;
+import org.apache.ignite.configuration.internal.annotation.Value;
+
+@Config(value = "test", root = true)
+public class TestConfigurationSchema {
+    @Value
+    private String value1;
+
+    @Value
+    private long primitiveLong;
+
+    @Value
+    private Long boxedLong;
+
+    @Value
+    private int primitiveInt;
+
+    @Value
+    private Integer boxedInt;
+}
```

Comment:
No empty line. And you forgot about comments

Reply:
This is a test java file in resources, I'm not sure if it should follow our style guides as we only need in the test's runtime.